packaging_automation/templates/docker/postgres-15/postgres-15.tmpl.dockerfile

```
@@ -0,0 +1,42 @@
+# This file is auto generated from it's template,
+# see citusdata/tools/packaging_automation/templates/docker/postgres-15/postgres-15.tmpl.dockerfile.
+FROM postgres:{{postgres_version}}
+ARG VERSION={{project_version}}
+LABEL maintainer="Citus Data https://citusdata.com" \
+      org.label-schema.name="Citus" \
+      org.label-schema.description="Scalable PostgreSQL for multi-tenant and real-time workloads" \
+      org.label-schema.url="https://www.citusdata.com" \
+      org.label-schema.vcs-url="https://github.com/citusdata/citus" \
+      org.label-schema.vendor="Citus Data, Inc." \
+      org.label-schema.version=${VERSION} \
+      org.label-schema.schema-version="1.0"
+
+ENV CITUS_VERSION ${VERSION}.citus-1
+
+# install Citus
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       ca-certificates \
+       curl \
+    && curl -s https://install.citusdata.com/community/deb.sh | bash \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-{{project_minor_version}}=$CITUS_VERSION \
+                          postgresql-$PG_MAJOR-hll=2.17.citus-1 \
+                          postgresql-$PG_MAJOR-topn=2.5.0.citus-1 \
```

```suggestion
                          postgresql-$PG_MAJOR-hll=2.18.citus-1 \
                          postgresql-$PG_MAJOR-topn=2.6.0.citus-1 \
```

These are the latest versions.

Maybe we should update similar files in the same directory as well.

---

Fixed this one